.github/ISSUE_TEMPLATE/BUG_REPORT.yml

```diff
@@ -0,0 +1,69 @@
+name: Bug Report
+description: Create a report to help us improve
+title: " "
+labels: ['type:Bug', 'status:Unconfirmed']
+body:
+  - type: textarea
+    id: "summary"
+    attributes:
+      label: Issue summary
+      description: "A brief summary of the issue"
+    validations:
+      required: true
+  - type: textarea
+    id: "steps"
+    attributes:
+      label: "Steps to reproduce"
+      value: |
+        1. Step 1
+        2. Step 2
+        3. ...
+    validations:
+      required: true
+  - type: input
+    id: "version-pyth"
+    attributes:
+      label: "Python version"
+      description: "Execute python --version"
+      placeholder: "Execute python --version"
```

Comment:
All the versions can be put into a single text area, just like done [here](https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/syntax-for-issue-forms#yaml-issue-form-template).

Put how to get the version number in the description itself.